prowler/providers/aws/services/eventbridge/eventbridge_bus_exposed/eventbridge_bus_exposed.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "aws",
+  "CheckID": "eventbridge_bus_exposed",
+  "CheckTitle": "Ensure that your AWS EventBridge event bus is not exposed to everyone",
+  "CheckType": [],
+  "ServiceName": "eventbridge",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
+  "Severity": "high",
+  "ResourceType": "AwsEventsEventbus",
+  "Description": "Ensure that your AWS EventBridge event bus is not exposed to everyone.",
+  "Risk": "If your AWS EventBridge event bus is exposed to everyone, unauthorized users can access your event bus and potentially view or modify your events.",
+  "RelatedUrl": "https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CWE_GettingStarted.html",
+  "Remediation": {
+    "Code": {
+      "CLI": "aws events remove-permission --statement-id aaaabbbb-aaaa-bbbb-cccc-1234567890123456789012345",
```

Comment:
Don't you need to include the resource id here?

Reply:
Done